apps/explorer/src/pages/transaction-result/programmable-transaction-view/TransactionsCard.tsx

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { type SuiTransaction } from '@mysten/sui.js';
```

Comment:
Can you take all of these components out of `pages` and put them into some folder in `components`?

Reply:
moved to `ui` root